importer.js

```diff
@@ -0,0 +1,56 @@
+import fs from "fs";
+import nodePath from "path";
+import csvjson from "csvjson";
+import { promisify } from "util";
+const readFileAsync = promisify(fs.readFile);
+const readDirAsync = promisify(fs.readdir);
+
+class Importer {
+    constructor(dirWatcher) {
+        dirWatcher.on('changed', (path) => {
+            /**
+             * Watching for file changes Syncrhonously
+             */
+            var result = this.importSync(path);
+            console.log(result)
+
+            /**
+             * Asynchronously watching the file changes
+             */
+            this.import(path).then((data) => {
```

Comment:
Since you have define `import` method as an `async` function, you can implement `await` syntax to execute this asynchronous method.

Reply:
Thanks for the advice. Updated the with async callback and await for import.